about.html

```diff
@@ -5,16 +5,27 @@
   <div class="col-sm-6">
     <h2>About VuFind®</h2>
 
-    VuFind® is a library resource portal designed and developed for libraries by libraries. The goal of VuFind® is to enable your users to search and browse through all of your library's resources by replacing the traditional OPAC to include:
+    <p>
+      VuFind® is a discovery system designed and developed for libraries by libraries, and also suitable for building
+      search interfaces to all kinds of content. The goal of VuFind® is to enable your users to search and browse
+      through all of your resources in a single consistent and user-friendly interface. Content could include:
+    </p>
     <ul>
-      <li>Catalog Records
-      <li>Digital Library Items
-      <li>Institutional Repository
-      <li>Institutional Bibliography
-      <li>Other Library Collections and Resources
+      <li>Catalog records</li>
+      <li>Institutional repository content</li>
+      <li>Locally hosted journal articles</li>
```

Comment:
I've changed this to "Open access journal articles" -- since that's really the use case we're trying to describe (e.g. OJS). I also added "Websites" (since that's a common use case we missed) and made an attempt to acknowledge licensed content with the caveat that it's not always possible. Maybe the list is too long now, but I thought I'd add before I removed. :-)